pibooth_picture_template.py

```diff
@@ -111,13 +111,11 @@ def parse(self):
             orientation = pictures.PORTRAIT if size[0] < size[1] else pictures.LANDSCAPE
 
             captures = self.parse_captures(template)
-            subdata = data.setdefault(orientation, {}).setdefault(len(captures), {})
-            if subdata:
-                raise TemplateParserError("Several templates with {} captures are defined".format(
-                    len(captures)))
-
+            _captures = []
+            count_captures = set()
             for capture in captures:
                 style = self.parse_style(capture)
+                count_captures.add(capture.get('value'))
```

Comment:
This is not retro compatible with previous version of the plugin where numbering was not mandatory